Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12 AS builder
+WORKDIR /go/src/github.com/openshift/machine-config-operator
+COPY . .
+RUN WHAT=machine-config-operator ./hack/build-go.sh
```

Comment:
Not required to do in this PR but I think as a followup we should just make `./hack/build-go.sh` build everything by default and do that in `RUN` to reduce duplication.

Reply:
happy to refactor as a followup PR :+1: